Test failures: UnicodeDecodeError in QuartairStratigrafie tests

There are test failures on AppVeyor regarding encoding in the QuartairStratigrafie XSD files. Travis seems to be OK.

Needs further investigation (could it be it only fails on Windows..?).

```text
tests\test_search_itp_quartairstratigrafie.py:198: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
pydov\search\interpretaties.py:910: in search
    return_fields=return_fields)
pydov\search\abstract.py:545: in _search
    self._pre_search_validation(location, query, return_fields)
pydov\search\abstract.py:418: in _pre_search_validation
    self._init_fields()
pydov\search\interpretaties.py:834: in _init_fields
    self._get_remote_xsd_schemas()
tests\test_search.py:325: in _get_remote_xsd
    data = f.read()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
self = <encodings.cp1252.IncrementalDecoder object at 0x000000FF19A38160>
input = b'<?xml version="1.0" encoding="UTF-8"?>\n<!-- edited with XMLSpy v2010 (http://www.altova.com) by DPK (DPK) -->\n<xs:...ntation>\n\t\t\t\t</xs:annotation>\n\t\t\t</xs:enumeration>\n\t\t</xs:restriction>\n\t</xs:simpleType>\n</xs:schema>\n'
final = True
    def decode(self, input, final=False):
>       return codecs.charmap_decode(input,self.errors,decoding_table)[0]
E       UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 21607: character maps to <undefined>
```

--- a/tests/test_search.py
+++ b/tests/test_search.py
@@ -1,5 +1,6 @@
 """Module grouping tests for the boring search module."""
 import glob
+from io import open
 
 import pytest
 
@@ -272,7 +273,7 @@ def mp_dov_xml(monkeymodule, request):
 
     def _get_xml_data(*args, **kwargs):
         file_path = getattr(request.module, "location_dov_xml")
-        with open(file_path, 'r') as f:
+        with open(file_path, 'r', encoding="utf-8") as f:
             data = f.read()
             if type(data) is not bytes:
                 data = data.encode('utf-8')
@@ -321,7 +322,7 @@ def mp_remote_xsd(monkeymodule, request):
         schemas = []
 
         for xsd_file in glob.glob(xsd_base_path):
-            with open(xsd_file, 'r') as f:
+            with open(xsd_file, 'r', encoding="utf-8") as f:
                 data = f.read()
                 if type(data) is not bytes:
                     data = data.encode('utf-8')

